karma.conf.js

```diff
@@ -30,7 +54,11 @@ module.exports = function (config) {
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ['kjhtml', 'progress'],
+    reporters: ['kjhtml', 'progress','failed-tests'],
+
+    failedTestsReporter: {
+      outputFile: 'test/karma/karma-failed-tests.txt',
+    },
```

Comment:
test failures will be saved here to be picked up by `npm run test:browser-ci:retry`.

Reply:
oh, that's pretty clever.  I wish we could do that with mocha